src/patternfly/demos/LoginPage/examples/login-page-simple-example.hbs

```diff
@@ -12,20 +12,97 @@
+          {{#> form }}
+            {{#> form-group}}
+              {{#> form-label form-label--attribute='for="login-demo-form-username"' required="true"}}Username{{/form-label}}
+              {{#> form-control controlType="input" form-control--attribute='required input="true" type="text" id="login-demo-form-username" name="login-demo-form-username" aria-describedby="login-demo-username-helper"'}}{{/form-control}}
+              {{#> form-helper-text form-helper-text--attribute='id="login-demo-username-helper" aria-live="polite"'}}Please provide a username{{/form-helper-text}}
+            {{/form-group}}
+            {{#> form-group}}
+              {{#> form-label form-label--attribute='for="login-demo-form-password"' required="true"}}Password{{/form-label}}
+              {{#> form-control controlType="input" form-control--attribute='required input="true" type="password" id="login-demo-form-password" name="login-demo-form-password" aria-describedby="login-demo-password-helper"'}}{{/form-control}}
+              {{#> form-helper-text form-helper-text--attribute='id="login-demo-password-helper" aria-live="polite"'}}Please provide a password{{/form-helper-text}}
+            {{/form-group}}
+            {{#> form-group form-group--modifier="pf-m-action"}}
+              {{#> toolbar}}
+                {{#> toolbar-group}}
+                  {{#> toolbar-item}}
+                    {{#> button button--modifier="pf-m-primary"}}
+                      Sign in
+                    {{/button}}
+                  {{/toolbar-item}}
+                {{/toolbar-group}}
+                {{#> toolbar-group toolbar-group--modifier="pf-u-mb-sm"}}
+                  {{#> toolbar-item}}
+                    {{#> check}}
+                      {{#> check-input check-input--attribute='type="checkbox" id="login-demo-checkbox" name="login-demo-checkbox"'}}{{/check-input}}
+                      {{#> check-label check-label--attribute='for="login-demo-checkbox"'}}Keep me signed in for 30 days.{{/check-label}}
+                    {{/check}}
+                  {{/toolbar-item}}
+                {{/toolbar-group}}
+              {{/toolbar}}
+            {{/form-group}}
+          {{/form}}
         {{/login-box-body}}
         {{#> login-box-footer}}
-          Login box footer. Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure labore ipsa nemo nostrum id culpa atque nulla expedita at! Explicabo aut minima alias. Officia magnam est at omnis iste eius?
+          <p class="pf-u-mb-md">Or log in with</p>
+          {{#> list list--modifier="pf-m-grid"}}
+            <li>
+              {{#> display display--type="flex" display--modifier="pf-u-align-items-center pf-u-mt-sm pf-u-mb-sm"}}
+                <img src="/assets/images/logo-google.svg" alt="Google logo" />
+                <p class="pf-u-ml-sm">Google</p>
```

Comment:
Are these list items meant to be links or buttons? Is the expectation that I could click one and see an alternate method of authenticating?

Reply:
The visual mockup didn't specify, but I imagine these would be SSO links that would take the user to the provider's site for authentication. I've updated the list items accordingly and tested them using VoiceOver. Do you think the VoiceOver readout should be more descriptive about the nature of this collection of links? Or is the "Or log in with" <p> directly preceding it enough of a hint?

Reply:
Oooh, that's a good question. Maybe using a heading tag, like `<h2>` is sufficient in this case.